packages/jest-config/src/normalize.js

```diff
@@ -154,23 +156,28 @@ const normalizeUnmockedModulePathPatterns = (
   return config[key].map(pattern =>
     utils.replacePathSepForRegex(
       pattern.replace(/<rootDir>/g, config.rootDir),
-    )
-  );
+    ));
 };
 
 const normalizePreprocessor = (config: InitialConfig) => {
   /* eslint-disable max-len */
   if (config.scriptPreprocessor && config.transform) {
     throw createConfigError(
-`  Options: ${chalk.bold('scriptPreprocessor')} and ${chalk.bold('transform')} cannot be used together.
-  Please change your configuration to only use ${chalk.bold('transform')}.`
+      `  Options: ${chalk.bold('scriptPreprocessor')} and ${chalk.bold(
+        'transform',
+      )} cannot be used together.
```

Comment:
We still don't really know how to format those. https://github.com/prettier/prettier/pull/749 is an attempt where it removes all the \n so that it actually fits in one line again.